bin/wick-run

```diff
@@ -17,6 +17,12 @@ wickRun() {
         return 1
     fi
 
+    # Set up the KV store. If Wick gets split so the prep work can be done
```

Comment:
The situation it is describing is when Wick is split. Splitting means half runs on the destination and half runs on the source machine. Maybe a comma, a word, or rephrasing is necessary but omission of the word starts to imply that the prep work can get done when Wick gets split, and that's inaccurate.

745c661